### PR TITLE
Jy2/romania

### PR DESCRIPTION
**Pull request by jy2wong**

### CODE REVIEW

**Review by gyp — APPROVED** [review]
Looks good to me from the translation point of view. As we start adding more and more sources for local data points, I'd consider hiding the source list behind a popup or something, but it's OK for now.

File: update_prevalence.py [hunk 1]
```diff
@@ -730,6 +742,13 @@ def main() -> None:
                         line.Seven_day_smoothed_daily_change * 7
                     )
 
+        # Romanian county (judet) data. Treat as states internally.
```
Comment (beshaya) [inline update_prevalence.py:745]: I think you want Romania to be the State and the judet to be the County? Unless you expect sub-county data in the future...

(This is similar to how the UK works)

Comment (jy2wong) [reply update_prevalence.py:745]: https://en.wikipedia.org/wiki/Administrative_divisions_of_Romania
https://en.wikipedia.org/wiki/Jude%C8%9B
A județ (Romanian county) is the largest administrative division after "entire country", which is why I classified it as a "state" in update_prevalence.py.

**Comment by jy2wong** [comment]
Removed the prevalence data roll, @blanchardjeremy !

**Review by blanchardjeremy — APPROVED** [review]
LGTM

**Comment by beshaya** [comment]
> A județ (Romanian county) is the largest administrative division after
"entire country", which is why I classified it as a "state" in
update_prevalence.py.

Sure, but the help text says "US State or Country" so this isn't very
discoverable. Possibly this indicates a need for 3 levels of selection
(country, state, county) to be equitable between countries, in which case
setting a Judet to be a state level item would make sense.

From a precision perspective, Romania is about the size of a large-ish US
state, which is perhaps a after-the-fact justification for the hierarchy
that previously existed.

**Comment by jy2wong** [comment]
I think people generally think of Romania as a country, and the secondary level is called "region" in the location selection interface.

If the distinction between country/state/county is size-based, then that needs documenting. I interpreted country as the largest geographical area you would use on a passport to describe your place of birth, and state as the next largest division after country with something that could plausibly be called a separate governing body (and hence its own separate prevalence data).

I interpreted "county" as something that really only applies to a place as big in physical size as the US/Canada or population as India/China where there's still a lot of variance within a "state".

**Comment by beshaya** [comment]
Oh, wait, this ends up doing the thing I think it should. The wording in
update prevalence is just a bit confusing.

**Pull request merged**